pulsar/src/tests.rs

```diff
@@ -175,3 +177,48 @@ fn custom_log_dir_test() {
     #[cfg(target_os = "windows")]
     assert!(log_path.ends_with("AppData/Local/pulsar/logs"));
 }
+
+#[cfg(test)]
+mod create_or_move_data_tests {
```

Comment:
You can just have `mod test`

Reply:
Actually, the aim is to keep the related tests under one mod label.
Let me know your thoughts.